### PR TITLE
feat: make up the token manager pallet

### PR DESCRIPTION
**Pull request by energyGiver**
### Summary
* make up the token manager pallet from the skeleton version
* It's the simple version to test, features will be implemented delete-asset-link, update-asset-link and so on in the near future

### Describe your changes
* SystemTokenTable: this storage map store the link between parachain AssetId and relay chain AssetId
```rust
pub(super) type SystemTokenTable<T: Config> =
		StorageMap<_, Twox64Concat, (ParachainId, ParachainAssetId), RelayChainAssetId>;
```

* insert_asset_link: this function should be called by root_account(Alice)
    - insert the link between parachain AssetId and relay chain AssetId
    - not use the function name like `asset_register` to avoid a confusion for `AssetRegistry`
```rust
pub fn insert_asset_link(
			origin: OriginFor<T>,
			para_id: ParachainId,
			para_asset_id: ParachainAssetId,
			relay_asset_id: RelayChainAssetId,
		)
```

### Related Issue
* #30 

### Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] If it is a core feature, I have added thorough tests.
- [x] Well documented code
- [x] `cargo clippy`
- [x] `cargo-nightly fmt`

### CODE REVIEW

File: bin/node-template/runtime/src/lib.rs [hunk 1]
```diff
@@ -352,7 +355,8 @@ construct_runtime!(
 		Assets: pallet_assets,
 		Authorship: pallet_authorship,
 		TransactionPayment: pallet_transaction_payment,
-		InfraAssetTxPayment: pallet_infra_asset_tx_payment,
+		// InfraAssetTxPayment: pallet_infra_asset_tx_payment,
```
Comment (sweatpotato13) [inline bin/node-template/runtime/src/lib.rs:358]: 이건 왜 주석처리 하신건가요?

**Comment by cocoyoon** [comment]
crate 명 `infra-*` 통일 부탁드립니다
```
- infra-system-token-manager
- infra-system-token
- infra-token-manager
```
개인적으로는 `system` 이 들어갔으면 좋겠습니다

File: bin/node-template/runtime/src/lib.rs (hunk 1 above)
Comment (energyGiver) [reply bin/node-template/runtime/src/lib.rs:358]: Due to InfraAssetTxPayment, couldn't send extrinsic in the polkadot.js 😢

**Comment by energyGiver** [comment]
> crate 명 `infra-*` 통일 부탁드립니다
> 
> ```
> - infra-system-token-manager
> - infra-system-token
> - infra-token-manager
> ```
> 
> 개인적으로는 `system` 이 들어갔으면 좋겠습니다

Good, I think infra-system-token-manager is better, too

**Comment by energyGiver** [comment]
> crate 명 `infra-*` 통일 부탁드립니다
> 
> ```
> - infra-system-token-manager
> - infra-system-token
> - infra-token-manager
> ```
> 
> 개인적으로는 `system` 이 들어갔으면 좋겠습니다

resolved it!

**Review by cocoyoon — APPROVED** [review]

**Review by sweatpotato13 — APPROVED** [review]
👍

**Pull request merged**